src/Downloader.ts

```diff
@@ -175,13 +176,13 @@ class Downloader {
       this.baseUrl = basicURLDirector.buildDownloaderBaseUrl([
         { condition: await MediaWiki.hasWikimediaMobileApi(), value: MediaWiki.WikimediaMobileApiUrl.href },
         { condition: await MediaWiki.hasWikimediaDesktopApi(), value: MediaWiki.WikimediaDesktopApiUrl.href },
-        { condition: await MediaWiki.hasVisualEditorApi(), value: MediaWiki.visualEditorApiUrl.href },
+        // { condition: await MediaWiki.hasVisualEditorApi(), value: MediaWiki.visualEditorApiUrl.href },
```

Comment:
1 - create a PR to deactivate visualeditor
2 - create a ticket requesting reintroducing it once the phabricator ist fix and rolled out
3 - put links between all of this so things are understandable

Only the VisualEditor autodetection should be deactivated

Reply:
Done, this PR has been merged - https://github.com/openzim/mwoffliner/pull/1941
Ticket about reintroducing VE - https://github.com/openzim/mwoffliner/issues/1940